package.json

```diff
@@ -14,6 +14,8 @@
   },
   "devDependencies": {
     "@babel/core": "^7.7.0",
+    "@testing-library/jest-dom": "^5.12.0",
+    "@testing-library/react": "^11.2.6",
```

Comment:
We already have these libraries in [@theforeman/test](https://github.com/theforeman/foreman-js/blob/3e463939623016543474c93e9591d4129ee80d91/packages/test/package.json#L28)

Is this version bump require?